BikeData/src/main/scala/by/itechart/internship/GeneralStats.scala

```diff
@@ -0,0 +1,28 @@
+package by.itechart.internship
+
+import java.time.temporal.ChronoUnit
+
+object GeneralStats {
+  def parserGeneralStats(configValues: ConfigValues, table: List[Array[String]]): Unit = {
```

Comment:
This function does several things at once, and it's bad design pattern
What it does:
1) aggregates data
2) prepares output
3) writes output to the file

How 3) from GeneralStats  is different from the same code inside UsageStats? Try to reuse this logic